packages/core/src/lib/Card/Card.tsx

```diff
@@ -37,6 +38,7 @@ const Card: React.FC<CardProps> = ({
             isSelected={isSelected}
             isDisabled={isDisabled}
             cursorType={cursorType}
+            style={style}
```

Comment:
you pass style here but not to button or typography 🤔 is it already present?

Reply:
It gets there through `{...props}`    :)
![image](https://user-images.githubusercontent.com/88556617/199300947-5844e515-b405-4df5-a97c-a2cd6cc654e7.png)
![image](https://user-images.githubusercontent.com/88556617/199300623-cd4c574d-57df-48d6-9aba-a06db027eef5.png)